### PR TITLE
fix: lunary callback tags

### PR DESCRIPTION
**Pull request by hughcrt**
## Title

Fix a bug where tags where not properly reported to the Lunary integration

## Type

🐛 Bug Fix

## Changes

The current test pipeline failed before my changes. But I've changed only on line, which won't have any effect on the tests.

### CODE REVIEW

**Comment by ghost** [comment]
hey @hughcrt is there a test we could add for this, here - https://github.com/BerriAI/litellm/blob/7c23077a71726e183610249303d231ca9aae541d/litellm/tests/test_lunary.py#L14

would help prevent future regressions 

for reference you can see how we do unit tests for langfuse
https://github.com/BerriAI/litellm/blob/7c23077a71726e183610249303d231ca9aae541d/litellm/tests/test_alangfuse.py#L281

**Comment by hughcrt** [comment]
Hi @krrishdholakia, I will add that tomorrow. In the meantime, would it be possible to merge this PR? Some of litellm users that are also our customer have unexpected behavior because of that

File: litellm/integrations/lunary.py [hunk 1]
```diff
@@ -105,19 +105,17 @@ def log_event(
...
             optional_params = kwargs.get("optional_params", {})
-            metadata = litellm_params.get("metadata", {}) or {}
+            metadata = litellm_params.get("metadata", {})
 
             if optional_params:
-                # merge into extra
                 extra = {**extra, **optional_params}
 
-            tags = litellm_params.pop("tags", None) or []
+            tags = metadata.get("metadata", None)
```
Comment (ghost) [inline litellm/integrations/lunary.py]: there's no metadata inside metadata, this will always be None 

is this intended behaviour? @hughcrt

Comment (hughcrt) [reply litellm/integrations/lunary.py]: sorry, I've pushed the wrong thing. It's fixed in the next commit

**Comment by hughcrt** [comment]
Hi @krrishdholakia, I've added the tests as you ask. Could you please merge the PR? Thanks in advance.

**Comment by hughcrt** [comment]
Hi @krrishdholakia, any update?

**Comment by ghost** [comment]
Missed this. thanks for the work here @hughcrt

**Pull request merged**